lib/Service/ExAppConfigService.php

```diff
@@ -65,6 +84,9 @@ public function setAppConfigValue(string $appId, string $configKey, mixed $confi
 				return null;
 			}
 		}
+		if ($sensitive) {
+			$appConfigEx->setConfigvalue($configValue);
```

Comment:
A comment would be nice here, something like "Setting the plain text value in the object for local usage".